internal/users/tests/tests.go

```diff
@@ -97,4 +109,5 @@ func Mockgpasswd(_ *testing.T) {
 		fmt.Fprintf(os.Stderr, "Mock: error while writing in file: %v", err)
 		os.Exit(1)
 	}
+	f.Close()
```

Comment:
good catch!

Reply:
Yeah, but it is still wrong :joy: I'll fix it in the next commit